spec/Model.spec.js

```diff
@@ -622,15 +622,13 @@ describe('Model', () => {
     it('returns data in valid jsonapi structure with coerced values', async () => {
       const todo = store.add('organizations', { id: 1, title: 'Buy Milk' })
       expect(todo.jsonapi()).toEqual({
-        data: {
```

Comment:
Whoa, why are we changing the shape here and promoting the `data` properties to the top? Isn't this incompatible with the spec? How is this not a breaking change?

Reply:
`jsonapi` is a function that we wrote to help convert the data to the appropriate shape for the payload. The spec is concerned with the final request. We compensate for this change [here](https://github.com/artemis-ag/mobx-async-store/pull/52/files?file-filters%5B%5D=.js#diff-96402e3967f40f890b345d1291b004b3R285) by adding `data` back in while constructing the body.

This lets us reuse this method to build data for multiple targets (see [bulkSave](https://github.com/artemis-ag/mobx-async-store/pull/52/files?file-filters%5B%5D=.js#diff-96402e3967f40f890b345d1291b004b3R285)) and set the `data` of the payload to an array of those transformed objects when making a bulk request.